tests/testcases.py

```diff
@@ -19,126 +19,165 @@ def __repr__(self):
         return "ReturnNone"
 
 
-today = datetime.strptime(datetime.now(tz=timezone("Europe/Berlin")).strftime("%Y-%m-%d %H:%M:%S"), "%Y-%m-%d %H:%M:%S")
+class Expected:
+    zone = timezone("Europe/Berlin")
```

Comment:
The class constant doesn't make any sense because it's only being used once (the line below)